Models/RequestBodies/ServiceCreate.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace PSP.Models
```

Comment:
should be under PSP.Models.RequestBodies namespace

Reply:
also, I still haven't looked through the entire PR, but RequestBodies sounds a lot like the DTO pattern, maybe the namespace could be renamed to `PSP.Models.Dtos` or smth

also also, I think it would be good to put the dtos or RequestBodies (whichever you decide to go with) to separate namespace/location than Models, not under it (i.e. `PSP.Dtos`) - it would bring a bit of clarity, since models as far as I know are usually associated with db tables/entities, so it feels slightly weird to see dtos here